cli_test/helpers/fixtures.go

```diff
@@ -0,0 +1,75 @@
+package helpers
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
```

Comment:
could you run `make format` for these files.

Reply:
We should create a git pre-commit hook for `make format`. I'll do it as soon as I spare few mins.